packages/nds-forms/scss/checkbox.scss

```diff
@@ -0,0 +1,101 @@
+@import "~@nice-digital/nds-core/scss/core";
+
+$checkbox-size: 32px;
+
+.checkbox {
+  @include font-size(p);
+  display: block;
+  position: relative;
+  min-height: $checkbox-size;
+  margin-bottom: rem($spacing-medium);
+  padding-left: $checkbox-size * 1.4;
```

Comment:
This 1.4 is a bit of a magic number - can we use one of the spacing vars instead?